### DISCUSSION TITLE
连接重置问题

### DISCUSSION THREAD

**Discussion by LogicCoder** [kbengine/kbengine#1411] [General]
TCPPacketReceiver::processPendingEvents(xxx.xxx.22.123:46390): Throwing REASON_GENERAL_NETWORK - Connection reset by peer

这种情况是什么原因呢？

**Comment by Amither113** [comment]
我也遇到的同样的情况，并成功解决。
我是在docker中部署kbengine服务器，端口映射时将baseapp的20005udp端口映射成了20005tcp端口，因此当client对20005端口发送udp包时，协议不同导致连接充值。
